Documentation/project-docs/writing-tests.md

```diff
@@ -0,0 +1,93 @@
+This document provides an overview of paradigm we are using in our tests.
+
+# RemoteExecutor
```

Comment:
> Attaching a debugger when this is ran in UWP shows that it's referencing mscorlib which I thought we don't have in CoreFX.

Maybe the one that got built in some specific configuration was referencing mscorlib.  But when I look at the ones I have locally used during corefx/coreclr test runs, e.g.
C:\Users\stoub\Source\Repos\corefx\bin\AnyOS.AnyCPU.Debug\RemoteExecutorConsoleApp\netstandard\RemoteExecutorConsoleApp.exe
it's not referencing mscorlib.dll:
![image](https://cloud.githubusercontent.com/assets/2642209/26082381/895fe332-399d-11e7-91cd-ae6e99254604.png)